### PR TITLE
Feat/260/cloud domains crawler

### LINKED ISSUE #260: cloud domains crawler
Related to #247

### PR DESCRIPTION
**Pull request by sudiptob2**
closes #260

### CODE REVIEW

File: test/datastore_kinds
Comment (mshudrak) [inline test/datastore_kinds:1]: Thanks for the PR. Looks great. Is there any reason you are not including crawler test for the test domain or am I missing something?

**Review by mshudrak — COMMENTED** [review]
I have one question.

File: test/datastore_kinds
Comment (mshudrak) [reply test/datastore_kinds:1]: ah, I see. It is still draft. NVM, please ignore.

Comment (sudiptob2) [reply test/datastore_kinds:1]: Thanks much for the proactive review though ❤️

**Review by 0xDeva — APPROVED** [review]

**Pull request merged**